test/system/static.test.ts

```diff
@@ -495,3 +495,48 @@ test("static and dynamic scanning results are aligned", async (t) => {
   );
   // TODO: imageLayers is completely different
 });
+
+test("static scanning NGINX with dockerfile analysis matches expected values", async (t) => {
+  const thisIsJustAnImageIdentifierInStaticAnalysis = "nginx:latest";
+  const dockerfilePath = path.join(
+    __dirname,
+    "../fixtures/dockerfiles/library/nginx/Dockerfile",
+  );
+  const pluginOptionsWithDockerSave = {
+    staticAnalysisOptions: {
+      imagePath: getFixture("docker-save/nginx.tar"),
+      imageType: ImageType.DockerArchive,
+    },
+  };
+
+  const pluginResultStatic = await plugin.inspect(
+    thisIsJustAnImageIdentifierInStaticAnalysis,
+    dockerfilePath,
+    pluginOptionsWithDockerSave,
+  );
+
+  const results = pluginResultStatic.scannedProjects[0].depTree.docker;
```

Comment:
how about adding a test & assertion: "Dockerfile analysis results are identical between static and dynamic scans".
is that reasonable or completely redundant? from an API point of view

Reply:
Both dynamic and static scanning use the same calls to build the Dockerfile analysis and to build the response, so tbh I am not sure if there is value in comparing them?